tests/mock_crust/churn.rs

```diff
@@ -187,16 +227,12 @@ fn request_during_churn_node_to_group() {
 
         poll_and_resend(&mut nodes, &mut []);
 
-        // This puts the members of the dst group to the beginning of the vec.
-        sort_nodes_by_distance_to(&mut nodes, dst.name());
-
-        let num_received = nodes.iter()
-            .take(MIN_GROUP_SIZE)
-            .filter(|node| did_receive_get_request(node, src, dst, data_id, message_id))
-            .count();
+        let (quorum, num_received) = quorum_after(&nodes, src, dst, data_id, message_id);
```

Comment:
Yes, but if a split happens, the `quorum_before` is larger. The point is that the message could be received _before_ or _after_ a _merge_ or _split_. So all we can safely expect is that enough nodes receive it with respect to either the before or after group size.